src/optimize.jl

```diff
@@ -131,10 +144,6 @@ function MOI.get(model::Optimizer, attr::MOI.RawOptimizerAttribute)
 end
 
 function MOI.set(model::Optimizer, attr::MOI.RawOptimizerAttribute, value)
-    if attr.name == "num_solutions" && !(value isa Int && value >= 1)
-        msg = "value must be an `Int` that is >= 1"
-        throw(MOI.SetAttributeNotAllowed(attr, msg))
-    end
```

Comment:
I think it's best to keep this as-is. Removing the option is breaking

Reply:
sure, make sense